src/main/java/edu/ucsb/cs156/gauchoride/entities/Ride.java

```diff
@@ -0,0 +1,38 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+// import io.swagger.annotations.ApiModelProperty;
+
+import javax.persistence.GeneratedValue;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "ride")
+public class Ride {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private String student;
+
+//   @ApiModelProperty(allowableValues = "Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday")
+  private String day;
+  
+  private String start;
+  private String end;
+
+  private String pickup;
+  private String dropoff;
```

Comment:
consider pickupLocation and/or dropoffLocation because otherwise someone might think these mean pickup and dropoff time, and that could lead to some weird bugs when half the code base thinks one thing, and half the code base thinks the other.  I've seen that happen.